cmd/service/utils.go

```diff
@@ -50,23 +53,35 @@ func prepareService(path string) (importedService *service.Service) {
 }
 
 func downloadServiceIfNeeded(path string) (newPath string, didDownload bool, err error) {
-	didDownload = false
 	newPath = path
 	if govalidator.IsURL(path) {
 		newPath, err = createTempFolder()
 		if err != nil {
 			return
 		}
 		err = gitClone(path, newPath, "Downloading service...")
-		didDownload = true
+		didDownload = err == nil
 	}
 	return
 }
 
-func gitClone(url string, path string, message string) (err error) {
+func gitClone(repoURL string, path string, message string) (err error) {
+	u, err := url.Parse(repoURL)
+	if err != nil {
+		return err
+	}
+	if u.Scheme == "" {
+		u.Scheme = "https"
+	}
+	ref := plumbing.Master
```

Comment:
what about when the default branch is different than `master`. In case of the core for example we use the `dev` as a default branch.
Maybe keep the ref empty would be a better solution like that we really keep the default behavior to take de default branch and not force the `master`